tardis/io/tests/data/csvy_test_config.yml

```diff
@@ -0,0 +1,77 @@
+# Example YAML configuration for TARDIS
+tardis_config_version: v1.0
+
+supernova:
+  luminosity_requested: 8.770 log_lsun
+  time_explosion: 16 day
+  distance: 8.32 Mpc
+
+# standard atomic data base; get it from the tardis-refdata repository
+atom_data: /home/mew488/Research/TARDIS/tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5
+
+csvy_model: csvy_full.csvy
+
+model:
```

Comment:
how does that work? what takes precedent?

Reply:
How the model is loaded is handled by tardis/simulation/base.py in the from_config() class method. Currently, if the config has the 'csvy_model' keyword, then that is how the model is loaded.  I think this should actually be handled in the schema, where the user is only allowed to define either 'model' or 'csvy_model'

Reply:
raise error if both are defined.